### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by paulzay**
### What does this Pr do?
Build tests for tictactoe game

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hi Team. 👋

You both have done quite well on this. 👍

However, let's fix this:

- [x] Kindly include instructions in your readMe to run tests.
- [ ] Please use only one `expect` per `it` block in your tests, presently some `it` blocks have two `expect` cases.
- [ ] Consider renaming the branch name into a more descriptive name.

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

File: lib/game.rb [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-
+  @@counter = 0
   def initialize(player1, player2, player_choice, game_board)
     @player1 = player1
     @player2 = player2
     @player_choice = player_choice.player_choice
     @game_board = game_board
     @not_the_end = true
-    @@counter = 0
+    # @@counter = 0
```
Comment (mikenath223) [inline lib/game.rb:11]: - [ ]  Please remove this commented piece of code, if you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)) .

File: spec/game_spec.rb [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+  describe Board do
+    let(:player1) { Players.new('Paul') }
+    let(:player2) { Players.new('Zay') }
+    let(:board) { Board.new(player1, player2) }
+
+    describe '#initialize' do
+      it 'should accept player names as args' do
+        board
+      end
```
Comment (mikenath223) [inline spec/game_spec.rb]: - [ ] This serves no purpose please remove this `it` block, as there is no `expect` case in it.

**Review by trekab — APPROVED** [review]
## Approved! :tada: 
Well done, team! You have met the project requirements. :+1: You may proceed to your next project.

_Happy coding!_ :wink: 
__Treasure Kabareebe__
[GitHub](https://github.com/trekab) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

**Pull request merged**